Libraries/CMSIS/Device/Maxim/MAX32657/Source/system_max32657.c

```diff
@@ -121,31 +122,42 @@ __weak int Board_Init(void)
  */
 __weak void SystemInit(void)
 {
-    /* Configure the interrupt controller to use the application vector table in */
-    /* the application space */
-#if defined(__CC_ARM) || defined(__GNUC__)
-    /* IAR sets the VTOR pointer incorrectly and causes stack corruption */
+#if defined(__VTOR_PRESENT) && (__VTOR_PRESENT == 1U)
     SCB->VTOR = (uint32_t)__isr_vector;
-#endif /* __CC_ARM || __GNUC__ */
+#endif /* __VTOR_PRESENT check */
 
-    /* Make sure interrupts are enabled. */
-    __enable_irq();
+#if (__FPU_PRESENT == 1U)
+    /* Enable FPU - coprocessor slots 10 & 11 full access */
+    SCB->CPACR |= SCB_CPACR_CP10_Msk | SCB_CPACR_CP11_Msk;
+#endif /* __FPU_PRESENT check */
 
-    /* Enable instruction cache */
-    MXC_ICC_Enable(MXC_ICC);
```

Comment:
ICC enable removed?

Reply:
Yep, I didn't want to add it for the initial pass for the verification and roadtest teams. It is a new internal cache IP (well a superset of our existing one). We can add it for increased speedups, but we should do some testing for this new IP before we enable it by default as there have been some cache issues in the past.

Reply:
Ok, maybe comment it out and add a TODO so we don't forget?

Reply:
Good call.